src/elements/Notification/notification.less

```diff
@@ -0,0 +1,44 @@
+/**
+ * Notification styles
+ */
+
+@ui-notification-background-color: var(--ui-notification-background-color, var(--ui-color-gray-lighter));
```

Comment:
it's probably a good idea to add hard fallback values in the nested var()'s here

`var(--ui-notification-background-color, var(--ui-color-gray-lighter, #f3f3f3))`